code/modules/mob/living/carbon/human/human_helpers.dm

```diff
@@ -335,6 +335,8 @@
 		fitness_modifier *= 2
 	if (HAS_TRAIT(src, TRAIT_STRENGTH))
 		fitness_modifier *= 1.5
+	if (HAS_TRAIT(src, TRAIT_ROD_SUPLEX))
```

Comment:
is this intentional? you already gave them TRAIT_STRENGTH

Reply:
yes, it's intentional.